File: DSA/ARARTIK_TYAGI_590011633/DAY_16_QUESTION_2.py
def count_occurrences(arr, target):
    count = 0
    for value in arr:
        if value == target:
            count += 1
    return count

# User input
arr = list(map(int, input("Enter array elements separated by spaces: ").split()))
target = int(input("Enter target to count: "))
result = count_occurrences(arr, target)
print("Output:", result)

File: DSA/khushichaturvedi_590012789/Day_16_question_2.c
#include <stdio.h>

int countOccurrences(int arr[], int n, int target) {
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (arr[i] == target)
            count++;
    }
    return count;
}

int main() {
    int arr[] = {3, 5, 2, 3, 7, 3, 4};
    int n = sizeof(arr) / sizeof(arr[0]);
    int target = 3;

    int result = countOccurrences(arr, n, target);

    printf("Target %d appears %d times\n", target, result);

    return 0;
}
